examples/Api.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import metadata from 'component-metadata!react-big-calendar/Calendar';
-import transform from 'lodash/object/transform';
+import { transform } from 'lodash/object';
 
```

Comment:
this will import all lodash.object, whereas the original code only imports transform, so the bundle size is going to increase dramatically here.